.gitignore

```diff
@@ -11,6 +11,8 @@ Data/*.png
 
 # Test data (if missing will be downloaded from NeuroVault)
 test/data
+test/MOxUnit
+test/octave-statistics
```

Comment:
Can we remove this from .gitignore, now that the code is not depending anymore on octave-statistics?

Reply:
Whoops, yes this is now done!